docs/content/Deployment/Platform-Kubernetes.md

```diff
@@ -0,0 +1,516 @@
+          ports:
+            - containerPort: 3000
+            - containerPort: 4000
+          resources: {}
+          volumeMounts:
+            - mountPath: /cube/conf
+              name: cube-api-hostpath0
+      restartPolicy: Always
+      volumes:
+        - hostPath:
+            path: /home/docker/conf
+          name: cube-api-hostpath0
+status: {}
+```
+
+We'll also create a corresponding service for this deployment in a file called
+`cube-api-service.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    io.kompose.service: cube-api
+  name: cube-api
+spec:
+  ports:
+    - name: '3000'
+      port: 3000
+      targetPort: 3000
+    - name: '4000'
+      port: 4000
+      targetPort: 4000
+  externalIPs:
+    - 192.168.99.103
+  selector:
+    io.kompose.service: cube-api
+status:
+  loadBalancer: {}
+```
+
+Now that we've configured our Cube.js API, let's also set up a deployment for a
+Refresh Worker in `cube-refresh-deployment.yaml`:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: cube-refresh-worker
+  name: cube-refresh-worker
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: cube-refresh-worker
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        io.kompose.service: cube-refresh-worker
+    spec:
+      containers:
+        - env:
+            - name: CUBEJS_API_SECRET
+              value: secret
+            - name: CUBEJS_CUBESTORE_HOST
+              value: 'cubestore-router'
+            - name: CUBEJS_CUBESTORE_PORT
+              value: '3030'
+            - name: CUBEJS_DB_CLICKHOUSE_READONLY
+              value: 'true'
+            - name: CUBEJS_DB_HOST
+              value: github.demo.trial.altinity.cloud
+            - name: CUBEJS_DB_NAME
+              value: default
+            - name: CUBEJS_DB_PASS
+              value: demo
+            - name: CUBEJS_DB_PORT
+              value: '8443'
+            - name: CUBEJS_DB_SSL
+              value: 'true'
+            - name: CUBEJS_DB_TYPE
+              value: clickhouse
+            - name: CUBEJS_DB_USER
+              value: demo
+            - name: CUBEJS_DEV_MODE
+              value: 'true'
+            - name: CUBEJS_REDIS_URL
+              value: redis://redis:6379
+            - name: CUBEJS_REFRESH_WORKER
+              value: 'true'
+          image: cubejs/cube:v0.28.14
+          name: cube-refresh-worker
+          resources: {}
+          volumeMounts:
+            - mountPath: /cube/conf
+              name: cube-refresh-worker-hostpath0
+      restartPolicy: Always
+      volumes:
+        - hostPath:
+            path: /home/docker/conf
+          name: cube-refresh-worker-hostpath0
+status: {}
+```
+
+### Create Cube Store Router and Worker nodes
+
+With our Cube.js API and Refresh Worker set up, we can now begin setting up Cube
+Store. Create a new file called `cubestore-router-deployment.yaml`:
```

Comment:
I made some significant edits to these sample files. Here's what's changed:

1. Deployments are now StatefulSets.
2. Only one StatefulSet and Service for Workers. Workers are scaled by increasing the `Replicas` value.
3. The Service for Workers is now Headless.

Check out these files to see what I mean.

- https://github.com/cube-js/cube.js/blob/54009227e05fa327cda78936f00b2127a9225fa5/examples/kubectl/prod/cubestore-router-statefulset.yaml
- https://github.com/cube-js/cube.js/blob/54009227e05fa327cda78936f00b2127a9225fa5/examples/kubectl/prod/cubestore-router-service.yaml
- https://github.com/cube-js/cube.js/blob/54009227e05fa327cda78936f00b2127a9225fa5/examples/kubectl/prod/cubestore-workers-statefulset.yaml
- https://github.com/cube-js/cube.js/blob/54009227e05fa327cda78936f00b2127a9225fa5/examples/kubectl/prod/cubestore-workers-service.yaml